### PR TITLE
Add some badges for convenience.

### PR DESCRIPTION
**Pull request by rexwangcc**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
 # Job Manager
 
-[![CircleCI](https://circleci.com/gh/DataBiosphere/job-manager/tree/master.svg?style=svg)](https://circleci.com/gh/DataBiosphere/job-manager/tree/master)
+![CircleCI branch](https://img.shields.io/circleci/project/github/DataBiosphere/job-manager/master.svg?label=Tests%20on%20Circle%20CI&style=flat-square)
+![Github](https://img.shields.io/badge/Supported%20Backends-cromwell%20%7C%20dsub-blue.svg?&style=flat-square)
```
Comment (cjllanwarne) [inline README.md]: This one I'm not so sure about (eg what does it give us?)

Comment (rexwangcc) [reply README.md]: Oh so this one does nothing but shows that JMUI is supporting both Cromwell and dsub... not sure if this has changed 🤔

**Comment by rexwangcc** [comment]
@cjllanwarne I have 2 options for you:

1. One badge to the "databiosphere"(which only has JMUI docker images at the moment, but can grow in the future)
Github

2. Three badges for all of the images respectively
Github
Github
Github

Which do you prefer?

**Comment by cjllanwarne** [comment]
@rexwangcc right now when I click on them they're just links to the image file - is it possible to make these link to dockerhub directly?

**Comment by rexwangcc** [comment]
@cjllanwarne They are linking to the dockerhub already. Github renders those badges in the comments in a weird way... If you click them and then click the badges in the poped new page, they will bring you to the right place :octocat:

**Comment by cjllanwarne** [comment]
Ok - in that case I should say I really like option (2) `Three badges for all of the images respectively`, and otherwise this looks good to me!

**Comment by rexwangcc** [comment]
@rsasch Merging this PR. I'm not familiar with the release process, so I'm not going to add anything about this PR into the changelog, please do that before next release, thanks!!

**Pull request merged**